crawler/crawler.py

```diff
@@ -165,10 +173,10 @@ def main():
     parser.add_argument(
         '--features',
         dest='features',
-        type=str,
-        default=defaults.DEFAULT_FEATURES_TO_CRAWL,
+        type=csv_list,
+        default=get_config()['general']['features_to_crawl'],
```

Comment:
These calls are going away on the next PR. All the config loading will be done before calling snapshot()